packages/wallets/src/components/Dropzone/Dropzone.tsx

```diff
@@ -0,0 +1,123 @@
+import React, {
+    CSSProperties,
+    DetailedHTMLProps,
+    InputHTMLAttributes,
+    ReactNode,
+    RefObject,
+    useCallback,
+    useState,
+} from 'react';
+import classNames from 'classnames';
+import { useDropzone } from 'react-dropzone';
+import CloseIcon from '../../public/images/close-icon.svg';
+import { IconButton, WalletButton, WalletText } from '../Base';
+import './Dropzone.scss';
+
+type TProps = {
+    buttonText?: ReactNode;
+    description?: ReactNode;
+    fileFormats?: NonNullable<Parameters<typeof useDropzone>[0]>['accept'];
+    height?: CSSProperties['height'];
+    hoverMessage?: ReactNode;
+    icon: ReactNode;
+    maxSize?: NonNullable<Parameters<typeof useDropzone>[0]>['maxSize'];
+    minHeight?: CSSProperties['minHeight'];
+    minWidth?: CSSProperties['minWidth'];
+    width?: CSSProperties['width'];
+};
+
+const Dropzone: React.FC<TProps> = ({
+    buttonText = 'Upload',
+    description,
+    fileFormats,
+    height,
+    hoverMessage = 'Drop file here',
```

Comment:
how do you handle translation for this?

Reply:
We havent decide on translation yet, but it can be handled inside the component later if we have translation. Only i need to know whether we can extract string from variable or not, might need to change it based on the decision for that